packages/api/cms-api/src/dam/files/dto/copyFiles.types.ts

```diff
@@ -0,0 +1,48 @@
+import { Type } from "@nestjs/common";
+import { Field, ObjectType } from "@nestjs/graphql";
+
+import { FileInterface } from "../entities/file.entity";
+
+export interface CopyFilesResponseInterface {
+    numberNewlyCopiedFiles: number;
+    numberAlreadyCopiedFiles: number;
+    inboxFolderId?: string;
+    mappedFiles: Array<MappedFileInterface>;
+}
+
+export interface MappedFileInterface {
+    rootFile: FileInterface;
+    copy: FileInterface;
+    isNewCopy: boolean;
+}
+
+export function createCopyFilesResponseType({ File }: { File: Type<FileInterface> }) {
+    @ObjectType()
+    class CopyFilesResponse {
+        @Field(() => Number)
+        numberNewlyCopiedFiles: number;
+
+        @Field(() => Number)
+        numberAlreadyCopiedFiles: number;
+
+        @Field(() => String, { nullable: true })
+        inboxFolderId?: string;
+
+        @Field(() => [MappedFile])
+        mappedFiles: Array<MappedFile>;
+    }
+
+    @ObjectType()
+    class MappedFile {
+        @Field(() => File)
+        rootFile: FileInterface;
+
+        @Field(() => File)
+        copy: FileInterface;
+
+        @Field(() => Boolean)
+        isNewCopy: boolean;
```

Comment:
You're currently not using this information for anything. What do you plan to use it for?

Reply:
I removed it